### DISCUSSION TITLE
Fetching error on freshly built NextJs app deployed in EC2

### DISCUSSION THREAD

**Discussion by webbahproject** [vercel/next.js#14477] [Help]
Had this error when i try to start the page on the newly freshly created NextJs app (9.4.4).

![image](https://user-images.githubusercontent.com/57855436/85356869-a772a980-b542-11ea-99eb-17c396bd0a17.png)

Seems like all the failed to fetch files were from the .next folder... but i could see it fetched _app, webpack and main from .next just fine. 

I had no idea why it failed like that... i could see all the files inside the .next folder.

This app runs inside docker in our EC2 btw. and also there are another 2 nextjs app deployed in it, under the same domain. Not sure if that helps in anything..

**Comment by IamLizu** [comment]
Do you have configured the build id? It is required for multi-server deployments. 

```
module.exports = {
  generateBuildId: async () => {
    // You can, for example, get the latest git commit hash here
    return 'my-build-id'
  },
}
```

Learn more: [https://nextjs.org/docs/api-reference/next.config.js/configuring-the-build-id](https://nextjs.org/docs/api-reference/next.config.js/configuring-the-build-id)

**Reply by webbahproject** [reply]
I did the above, delete the .next folder and restart next app again, but it didnt work. Still getting the same fetching error. 

I can deploy NextJs just fine on PHP7 Centos server.. only in EC2 (with Docker) that i encountered this issue. I can't seem to do any user event or console logging from my code.

**Comment by timneutkens** [comment]
Looks like you booted up the Next.js development server in EC2.

You can potentially look at using https://vercel.com

**Reply by webbahproject** [reply]
Is there anything special with EC2 that needs to be dealt with when deploying NextJs app in EC2? Because i can deploy to PHP7 (Centos) server just fine. Only in EC2 it came out with that error above.
